meteor-app/imports/ui/pages/workspace/charts/component.js

```diff
@@ -38,18 +38,30 @@ export default class ChartsPage extends React.Component {
     const target = event.currentTarget;
     const {
       updateFilterMin,
+      rangeMin,
+      filterMax,
```

Comment:
Are you sure `filterMax` is still available? Did you test the code before submitting the PR?